RGLServerPlugin/src/RGLServerPluginManager.cc

```diff
@@ -45,17 +45,42 @@ void RGLServerPluginManager::PostUpdate(
         const ignition::gazebo::UpdateInfo& info,
         const ignition::gazebo::EntityComponentManager& ecm)
 {
-    ecm.EachNew<>([&](const ignition::gazebo::Entity& entity)-> bool {
-        return RegisterNewLidarCb(entity, ecm);});
+    ecm.EachNew<>
+            ([this, &ecm](auto&& entity) {
+                return RegisterNewLidarCb(
+                        std::forward<decltype(entity)>(entity),
+                        std::forward<decltype(ecm)>(ecm));
```

Comment:
You should not have to std::forward here. You already have a reference to ecm, so you can just pass it directly.
+
Same in line 71 below.